core/modules/layout/plugins/access/user_compare_layout_access.inc

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Plugin to provide access if two user account contexts are the same.
+ */
+class UserCompareLayoutAccess extends LayoutAccess {
```

Comment:
I see the silly redundancy here with PHPCS requesting both a `@file` docblock and a class docblock. But I could see this being necessary in the event that namespaces/use statements are defined. In which case there would be lines between the `@file` docblock and the class docblock.

This was touched on in https://github.com/backdrop-ops/phpcs/issues/11

I'd suggest the `@file` docblock should simply be `Contains UserCompareLayoutAccess class.`